script/domVisitor.ts

```diff
@@ -54,14 +57,20 @@ async function visitPages(page: puppeteer.Page, bookPages: string[], audit: Audi
   });
 
   for (const pageUrl of bookPages) {
-    await page.goto(`${rootUrl}${pageUrl}${queryString ? `?${queryString}` : ''}`);
-    await page.waitForSelector('body[data-rex-loaded="true"]');
-    await calmHooks(page);
-
-    const matches = await page.evaluate(audit);
-    if (matches.length > 0) {
-      bar.interrupt(`- (${matches.length}) ${basename(pageUrl)}#${matches[0]}`);
+    try {
+      await page.goto(`${rootUrl}${pageUrl}${queryString ? `?${queryString}` : ''}`);
+      await page.waitForSelector('body[data-rex-loaded="true"]');
+      await calmHooks(page);
+
+      const matches = await page.evaluate(audit);
+
+      if (matches.length > 0) {
+        bar.interrupt(`- (${matches.length}) ${basename(pageUrl)}#${matches[0]}`);
+      }
+    } catch (e) {
```

Comment:
Is it possible to use the syntax without `(e)` here?

Reply:
no, its required even if you don't use it